### DISCUSSION TITLE
Integration One Wire?

### DISCUSSION THREAD

**Discussion by RMadosa** [jomjol/AI-on-the-edge-device#1156] [Q&A]
Hi
Is there any planning for an integration One Wire (e.g. DS18S20) or is there a site known where someone has already tried it based on this project?

Thx

**Comment by jomjol** [comment]
Hi,
this nobody has tried yet and it is not planned due to the following reasons:
1) very limited number of GPIOs available
2) only 3% of free RAM for the firmware --> use this limited amout for improving the core business of this device.

I would recommend to buy another 10 EUR device and place it next bye.
